org/chapter3.org

```diff
@@ -60,14 +60,14 @@
 சில உள்ளமைக்கப்பட்ட வகைகளின்(Builtin types) சில விளக்கங்களும் எடுத்துக்காட்டுகளும்
 இங்கே உள்ளன.
 
-| தரவுவகை | விளக்கங்கள்                       | எடுத்துக்காட்டு                           |
-|---------+---------------------------------+---------------------------------------|
-| void    | வெற்று வகை                      |                                       |
-| char    | ஒற்றை எழுத்து/பைட்               | char last_initial = 'H';              |
-| int     | முழுஎண்                         | int age = 23;                         |
-| long    | பெரிய மதிப்புகளை கொண்ட முழுஎண் | long age_of_universe = 13798000000;   |
-| float   | தசம(Decimal) எண்                | float liters_per_pint = 0.568f;       |
-| double  | அதிக துல்லியத்துடன் தசம எண்       | double speed_of_swallow = 0.01072896; |
+|   தரவுவகை  | விளக்கங்கள்                                                        | எடுத்துக்காட்டு                                                               |
+|---------+----------------------------------+---------------------------------------|
+| void    | வெற்று வகை                                                     |                                                                                        |
+| char    | ஒற்றை எழுத்து/பைட்                                       | char last_initial = 'H';              |
```

Comment:
This is just a source i plan to convert this to pdf or html so this won't be an issue.

Reply:
if some one tries read in org mode it will be usefull 😁